README.md

```diff
@@ -1 +1,5 @@
 # Tari Blockchain Explorer API
+
+## Development
+
+Use `NO_CRON=1 npm run start` to stop the cronjobs from running.
```

Comment:
Does it make sense to add `NO_CRON=` to `.env.sample`?

Reply:
It does.